### PR TITLE
RSDK-6488: Organize and stylize resource graph visualization.

### PR DESCRIPTION
**Pull request by dgottlieb**

### CODE REVIEW

File: resource/resource_graph.go [hunk 1]
```diff
@@ -119,40 +118,6 @@ func (g *Graph) Clone() *Graph {
 	return g.clone()
 }
 
-// ExportDot exports the resource graph as a DOT representation for visualization.
-// DOT reference: https://graphviz.org/doc/info/lang.html
-func (g *Graph) ExportDot() (string, error) {
```
Comment (dgottlieb) [inline resource/resource_graph.go:124]: After I looked at this from 10,000 feet, I felt I added enough to move the implementation to a separate file. Certainly not my default to split up a type and its member functions into different files.

File: robot/web/web_c.go [hunk 2]
```diff
@@ -384,19 +382,7 @@ func (svc *webService) handleVisualizeResourceGraph(w http.ResponseWriter, r *ht
 	if layout != "" {
 		gv.SetLayout(graphviz.Layout(layout))
 	}
-	img, err := gv.RenderImage(graph)
```
[the remark this answers is not in the document]
Comment (maximpertsov) [reply robot/web/web_c.go:387]: nah, svg is preferable - i reached for `RenderImage` since it was easier

[the remark this answers is not in the document]
Comment (maximpertsov) [reply robot/web/web_c.go:387]: actually, "easier" isn't even true - using `Render` would have been much more direct

**Review by maximpertsov — APPROVED** [review]

**Comment by maximpertsov** [comment]
This looks great!

**Pull request merged**